### PR TITLE
add a new evm compatible chain: okc exchain

### PR DESCRIPTION
**Pull request by ItsFunny**

### CODE REVIEW

File: okxchain/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "okxchain",
+  "assets": [
+    {
+      "description": "EVM compatible, Cosmos SDK based chain.",
```
Comment (JeremyParish69) [inline okxchain/assetlist.json:6]: This description should apply to the asset, not to the chain itself.

File: okxchain/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          "denom": "okt",
+          "exponent": 18
+        }
+      ],
+      "base": "wei",
+      "name": "OKXChain",
+      "display": "OKT",
+      "symbol": "OKT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/okxchain/images/okc.svg",
```
Comment (JeremyParish69) [inline okxchain/assetlist.json:22]: these images look more like images to be used for the chain, and if so, please use these images for the chain logo. Please instead point the asset's logo URIs to a token logo (would that be the Black OKX-square.png?).

File: okxchain/chain.json [hunk 3]
```diff
@@ -0,0 +1,108 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "okxchain",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "OKC",
```
Comment (JeremyParish69) [inline okxchain/chain.json:6]: Are we sure we want the chain's pretty name to be "OKC"? This is meant to be how people should say the name of the chain. It is my opinion that a user reading this is unlikely to know what this means, whereas something like "OKEx Chain" would be instantly recognizable.

File: okxchain/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "okt",
+          "exponent": 18
+        }
+      ],
+      "base": "wei",
+      "name": "OKXChain",
+      "display": "OKT",
```
Comment (JeremyParish69) [inline okxchain/assetlist.json:19]: should be lowercase. OKT doesn't not exist among the denom_units

File: okexchain/chain.json [hunk 5]
```diff
@@ -0,0 +1,108 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "exchain",
```
Comment (JeremyParish69) [inline okexchain/chain.json:3]: chain name in both files must match folder name. Folder: okexchain, assetlist.json: okxchain, chain.json: exchain. Please pick just one to use in all 3 places.

Comment (JeremyParish69) [reply okexchain/chain.json:3]: imo, 'exchain' would be harder to find in an alphabetized list than something starting with 'ok...', so I'd recommend either 'okxchain' or 'okexchain'

Comment (ItsFunny) [reply okexchain/chain.json:3]: Got it ,Thanks

**Pull request merged**